src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -355,7 +355,7 @@ public RESTZaak deleteBetrokkene(@PathParam("uuid") final UUID betrokkeneUUID,
 
     @POST
     @Path("zaak")
-    public RESTZaak createZaak(final RESTZaakAanmaakGegevens restZaakAanmaakGegevens) {
+    public RESTZaak createZaak(@Valid RESTZaakAanmaakGegevens restZaakAanmaakGegevens) {
```

Comment:
Just wondering: is there no way to enable validation automatically for all bean validation annotations? I think it's a bit of a shame to have to explicitly use @Valid to validate our Java beans. Not a big deal and maybe it's also ok like this. There is also something to be said to make it more explicit..